src/Microsoft.OData.Core/ODataBatchMimeReader.cs

```diff
@@ -0,0 +1,375 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ODataBatchMimeReader.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData.Core

```

Comment:
>Microsoft.OData. [](start = 10, length = 16)

Should probably create a "Mime" namespace (and corresponding folder) and put the Mime classes there.